test/powershell/Modules/ThreadJob/ThreadJob.Tests.ps1

```diff
@@ -261,7 +261,7 @@ Describe 'Basic ThreadJob Tests' -Tags 'CI' {
             Get-Job | where PSJobTypeName -eq "ThreadJob" | Remove-Job -Force
         }
 
-        $numThreadJobs = (Get-Job | where PSJobTypeName -eq "ThreadJob").Count
+        $numThreadJobs = @(Get-Job | where PSJobTypeName -eq "ThreadJob").Count
```

Comment:
Pester 4 syntax has `-HaveCount`. This can be converted to:
```PowerShell
@(Get-Job | where-object { PSJobTypeName -eq "ThreadJob"}) | Should -HaveCount 0
```

Reply:
Thanks.  Will change.